src/lib.rs

```diff
@@ -1975,6 +1993,7 @@ impl Connection {
                     let pkt = (b.buf()[..pkt_len]).to_vec();
 
                     self.undecryptable_pkts.push_back((pkt, *info));
+                    self.recv_bytes += pkt_len as u64;
```

Comment:
Oh, sorry, I missed this before, but this will cause the bytes of buffered packets to be counted twice, so this should probably be removed. Was it added on purpose?